unstructured/ingest/connector/sharepoint.py

```diff
@@ -202,6 +204,23 @@ def parent_name_matches(parent_type, permissions_filename, ingest_doc_filepath):
 
         return permissions_data
 
+    def _ensure_isoformat_datetime(self, timestamp: t.Union[datetime, str]) -> str:
```

Comment:
This is cool. If we end up needing it elsewhere we should put it in a utils file.

Reply:
Yep, totally agree!